frontend/src/modules/Shared/index.js

```diff
@@ -0,0 +1,9 @@
+export * from './FeedComments';
```

Comment:
Is it a comprehensive list of all `Shared` components or is it just what has been found so far? 
I am asking because if the list keeps growing (in the scope of modularization) we might need to think about reorganizing this into something like 
```
    Shared
           Comments
                  FeedCommentAdd
                  FeedComments
           Stack
                  Stack Stuff
           KeyValueTags
                  Key Value Stuff etc
```
And import it as `import { FeedComments } from 'modules/Shared/Comments';`
But it don't expect anything to pop up we can proceed